docs/design/collaboration/Building the collaboration paradigm between App Developers and Platform Developers with Kusion.md

```diff
@@ -0,0 +1,235 @@
+# Building the collaboration paradigm between App Developers and Platform Developers with Kusion
+
+# Context
```

Comment:
This is to better clarify the purpose of this design document:
We should **highlight** that we are discussing a practice or a proposed implementation (of the collaboration paradigm) under the **Kusion framework** (as opposed to we are discussing this paradigm itself).